app/views/users/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Update</h1>
+
+<%= form_for(@user) do |f| %>
+
+  <%= f.label :first_name %>
+  <%= f.text_field :first_name %>
+
+  <%= f.label :last_name %>
+  <%= f.text_field :last_name %>
+
+  <%= f.label :username %>
+  <%= f.text_field :username %>
+
+  <%= f.label :email %>
+  <%= f.email_field :email %>
+
+  <%= f.label :password %>
```

Comment:
Yeah like current password and new password, then only update if they are authenticated with the current password. I think I can make that work.